python/activity_stream/activity_stream.py

```diff
@@ -613,6 +613,10 @@ def show_new_note_dialog(self, modal=True):
 
         :param bool modal: Whether the dialog should be shown modally or not.
         """
+        if self._entity_id == None:
+            self._bundle.log_warning("Skipping New Note Dialog - No entity loaded.")
```

Comment:
I would probably make this a log_debug instead of a warning. I would consider this to be information that an artist/lead/sup using the app would need to see front and center.